jellyfin_kodi/jellyfin/http.py

```diff
@@ -198,7 +199,7 @@ def _get_header(self, data):
                 'Content-type': "application/json",
                 'Accept-Charset': "UTF-8,*",
                 'Accept-encoding': "gzip",
-                'User-Agent': self.config.data['http.user_agent'] or "%s/%s" % (self.config.data['app.name'], self.config.data['app.version'])
+                'User-Agent': self.config.data['http.user_agent'] or "%s/%s" % (self.config.data.get('app.name', 'Jellyfin for Kodi'), self.config.data.get('app.version', "0.0.0.0"))
```

Comment:
why '0.0.0.0'?  We only do 3 levels deep in our versioning.

Reply:
Too used to writing default route IP addresses?